src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -808,4 +808,16 @@ public void onCloseContainer(CallbackInfo ci) {
             mixinContainer.getCapturedTransactions().clear();
         }
     }
+
+    /**
+     * @author Faithcaio
```

Comment:
Add a date please. Otherwise, we don't know when and what version this was added.
Likewise, missing `@reason` before the description.